.github/build.yaml.gomplate

```diff
@@ -95,12 +109,22 @@
   {{{ $flavor := . }}}
   build-{{{ $flavor }}}:
     {{{ tmpl.Exec "runner" }}}
+    {{{ tmpl.Exec "needs" }}}
     env:
+      LUET_ARCH: {{{ $config.arch }}}
       FLAVOR: {{{ $flavor }}}
-      FINAL_REPO: {{{$config.organization}}}/{{{$config.repository}}}-{{{ $flavor }}}
+      ARCH: {{{ $config.arch }}}
+      FINAL_REPO: {{{$config.organization}}}/{{{$config.repository}}}-{{{ $flavor }}}{{{- if ne $config.arch "x86_64"}}}-{{{$config.arch}}}{{{end}}}
       DOWNLOAD_METADATA: false
       PUSH_CACHE: {{{$config.push_cache}}}
-      REPO_CACHE: {{{$config.organization}}}/{{{$config.cache_repository}}}-{{{ $flavor }}}-cache
+      REPO_CACHE: {{{$config.organization}}}/{{{$config.cache_repository}}}-{{{ $flavor }}}-cache{{{- if ne $config.arch "x86_64"}}}-{{{$config.arch}}}{{{end}}}
+      {{{- if has $config "luet_install_from_cos_repo" }}}
+      LUET_INSTALL_FROM_COS_REPO: {{{ $config.luet_install_from_cos_repo }}}
+      {{{- end }}}
+      {{{- if ne $config.arch "x86_64"}}}
+      # For non x86_64 archs, skip the *-fips build as there are no golang releases in other arches
+      SKIP_PACKAGES: "build/golang-fips toolchain-fips/yip toolchain-fips/luet-makeiso toolchain-fips/luet live/grub2 live/syslinux live/systemd-boot live/boot"
```

Comment:
A thought for a later iteration, maybe shall we include that as part of the `manifest.yaml` file as `skip_packages` for each arch? In that way we can control those more easily rather than relying on the CI

Reply:
yes, that is a good point. I might send an extra PR afterwards to move this there.